### ISSUE TITLE
Document has no main content area

### ISSUE THREAD

**Issue by LukasHirt** [owncloud/web#1652]

**Comment by pascalwengerter** [comment]
Closing this since we now have the Vue.js app wrapped in a `<main>` tag.

**Comment by marcus-herrmann** [comment]
But is it wrapping the main content of the page or everything?

**Comment by marcus-herrmann** [comment]
@pascalwengerter It looks like it is wrapping everything, and this is not how the main element is intended/could be a WCAG violation: https://github.com/owncloud/web/blob/695d3df37dcfa4a156ce02feaa0d28082fbb185f/packages/web-container/index.html.ejs#L28

**Comment by pascalwengerter** [comment]
@marcus-herrmann sorry, was under the wrong impression that [your commit](https://github.com/owncloud/web/commit/e196576db4ba239e88894433967388c08e3f7bf8) had fixed it. How critical is fixing this for the accessibility certification?

@kulmann maybe we need to be solving this on the extension level, currently we're using `<main>` in `web-container/index.html.ejs` _and_ `web-runtime/src/App.vue` (and from the definition below I'm not sure at which level we should be applying it, so are we wrapping extentions (e.g. draw-io) inside a `<main>` tag already or are they "bringing their own"?

> The content inside the <main> element should be unique to the document. It should not contain any content that is repeated across documents such as sidebars, navigation links, copyright information, site logos, and search forms.
> 
> Note: There must not be more than one <main> element in a document.

**Comment by marcus-herrmann** [comment]
>  How critical is fixing this for the accessibility certification?

@pascalwengerter Rather critical. Wrongly applied landmarks are worse than no landmarks at all. I'm quite certain this can be seen as a violation of WCAG SC 2.4.1, Bypass blocks.

My fix back in the day fixed this, but another `<main>` element wrapping it all, and one inside an app, wrapping the router is certainly difficult in an audit.
